components/AttemptSelector.vue

```diff
@@ -46,10 +46,17 @@ export default class AttemptSelector extends Vue {
   }
 
   get runAttempts(): Attempt[] {
```

Comment:
Maybe a few comments could be useful to clarify where each "filter" comes from, since there are quite a few here